custom/enikshay/tests/test_adherence_updates.py

```diff
@@ -144,15 +143,15 @@ def _create_episode_case(self, adherence_schedule_date_start, adherence_schedule
     def _create_adherence_cases(self, adherence_cases):
         self.factory.create_or_update_cases([
             get_adherence_case_structure(
-                adherence_date.strftime('%Y-%m-%d-%H-%M'),
+                "adherence{}".format(i),
```

Comment:
Because I want to test adherences that have same adherence_date and there is no time info, so they will get same case_id with previous name format, which means only one case will get created, so I name them to have unique names. The cases are cleaned up after every individual test case, so there is no collision.